content/doc/developer/internationalization/i18n-groovy-views.adoc

```diff
@@ -19,9 +20,9 @@ package jenkins.security.DownloadSettings
 
 def f = namespace(lib.FormTagLib)
 
-f.section(title:_("Plugin Manager")) {
-	f.entry(field: "useBrowser") {
-		f.checkbox(title: _("Use browser for metadata download"))
+f.section(title:_("title.pluginManager")) {
+	f.entry(field: "field.useBrowser") {
```

Comment:
This line surprises me.  It seems like it either needs the string wrapped in a call to gettext or it needs to not have the useBrowser reference.
```suggestion
	f.entry(field: _("field.useBrowser")) {
```

Have I misunderstood groovy form definition?

Reply:
Good catch. I've changed the call to respect gettext properly.